src/main/java/tc/oc/bingo/commands/CardCommand.java

```diff
@@ -87,6 +89,36 @@ public void bingoCardComplete(CommandSender sender, OnlinePlayer target, int ind
         .rewardPlayers(objectiveItem.getSlug(), Collections.singletonList(target.getPlayer()));
   }
 
+  @Subcommand("uncomplete")
+  @CommandPermission("bingo.uncomplete")
+  public void bingoCardUncomplete(
+      CommandSender sender, OnlinePlayer target, @Optional Integer index) {
+    if (!(Bingo.get().getBingoDatabase() instanceof MockDatabase)) {
+      sender.sendMessage("This command is only available when using a mocked database");
+      return;
+    }
+
+    Bingo.get()
+        .getPlayerCard(target.getPlayer().getUniqueId())
+        .whenComplete(
+            (c, t) -> {
+              if (Objects.isNull(c)) {
```

Comment:
Can you fix the Objects.isNull bits for actual == null etc

Reply:
This is a valid way of checking if an object is null

Reply:
It is but then you do it another way on line https://github.com/OvercastCommunity/Bingo/pull/7/files#diff-64654d382165b413ecab53ae2416bf6071a85df756e650474ee5323086972200R113 please stick with the == null != null format for consistency across the project.

Reply:
Changed format to objet == null